msgcntr/messageforums-app/src/webapp/css/msgcntr.css

```diff
@@ -2,7 +2,7 @@
 .forumHeader{
 	/*panel holding all forum metadata in forum list view*/
 	width: 100%;
-    background: #F8F8F3;
+    background: var(--sakai-background-color-2);
     border: 1px solid #D9C9A7;
```

Comment:
There's still a good amount of color, border-color, and background hardcoded in this file, please update all the values in the file to CSS Custom Props. If you can't reproduce the markup for more than a couple of them, it might be best to move onto another file and reassign this one to me or Shawn.